node/primitives/src/lib.rs

```diff
@@ -21,15 +21,19 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(not(feature = "std"), feature(alloc))]
 
-use runtime_primitives::generic;
-use runtime_primitives::{OpaqueExtrinsic, traits::BlakeTwo256};
+use runtime_primitives::{
+    generic, traits::{Verify, BlakeTwo256, Sr25519Signature}, OpaqueExtrinsic
```

Comment:
Why is `Sr25519Signature ` being imported from `traits` submodule of the runtime? Moving it outside (directly imported from `runtime_primitives`) seems to take the build error at least a step further toward success.

Reply:
fixed that - was just a typo.